### ISSUE TITLE
Feature-request: find-in-files for data manager 

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/eXide#106]
It would really help, if the find in files function would be included in the data manager ui. something like this. 
![screenshot 2015-10-03 11 39 58](https://cloud.githubusercontent.com/assets/6205362/10262385/476000e2-69c5-11e5-9fb5-135b8e014ab8.png)

**Comment by joewiz** [comment]
:+1:

**Comment by joewiz** [comment]
@duncdrum Just to confirm, you're aware that Fine in Files is available via eXide's Edit menu, right? 

> <img width="276" alt="screen shot 2017-03-12 at 5 09 56 pm" src="https://cloud.githubusercontent.com/assets/59118/23835967/ccbffd1c-0746-11e7-9c50-41bdfea15c7a.png">

> <img width="604" alt="screen shot 2017-03-12 at 5 10 00 pm" src="https://cloud.githubusercontent.com/assets/59118/23835969/d1ef747a-0746-11e7-9de8-9c57d4e73dcf.png">

So you're just requesting that a search field be added to the DB manager pane as in your illustration, right?

**Comment by duncdrum** [comment]
Yup, that's the idea. So you can do it when file manager windows is open, without having to switch windows.

**Comment by joewiz** [comment]
The find files dialog has several options - would these be exposed?

And how would the results be displayed - in the DH manager pane or a new one atop that?

**Comment by duncdrum** [comment]
@joewiz since I have no clue how file manager works right now it's hard to judge. I'd say fancy solution: do it inside file-manager window, ala Finder in macOS. Basic have the ``In: C[c?]ollection`` match the current collection open in file-manager, case and regular off (by default) and results display in the same way as what your 2nd screenshot would generate. 

Fancy macOS finder: If you type a search and hit return it filters the contents of the current ditectory based on that (in that file-manager window). If you click on the magnifier glass icon you get the advanced options (like in the current find in files pop-up).

Alternatively, highlighting file- or collection- names with hits within file-manager would work too. 

Anything that let's me check a collection of 100+ items for a string in the filename from the manager other then inspection by scrolling, would be a start.
